pandas/_libs/tslibs/nattype.pyx

```diff
@@ -14,10 +14,9 @@ cimport numpy as cnp
 from numpy cimport int64_t
 cnp.import_array()
 
-cimport util
-from util cimport (get_nat,
-                   is_integer_object, is_float_object,
-                   is_datetime64_object, is_timedelta64_object)
+cimport pandas._libs.tslibs.util as util
```

Comment:
The `cimports` weren't working when trying to use a relative import, so I've kept them as absolute